docs/instruments/NDB_BVL_Instrument_UPLOADER_TEMPLATE.class.inc

```diff
@@ -150,7 +150,7 @@ class NDB_BVL_Instrument_TEST_NAME extends NDB_BVL_Instrument
         }
         //echo error messages
         if(!empty($file->errorLog)){
-        while(list($fileType,$fileErrors)=each($file->errorLog)){
+        foreach($file->errorLog AS $fileType => $fileErrors){
```

Comment:
Please remove the `if` statement above. if it's empty, the foreach block will just run zero time.
Also fix indentation of curly brackets.